pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "meta-memcache"
-version = "1.0.5"
+version = "1.0.6"
```

Comment:
If this library is following semantic versioning, I think this would technically be `1.1.0` since there is a new API.